app/models/physical_server_provision_request.rb

```diff
@@ -0,0 +1,20 @@
+class PhysicalServerProvisionRequest < MiqProvisionConfiguredSystemRequest
+  TASK_DESCRIPTION  = 'Physical Server Provisioning'.freeze
+  SOURCE_CLASS_NAME = 'PhysicalServer'.freeze
+
+  def src_configured_systems
+    PhysicalServer.where(:id => options[:src_configured_system_ids])
+  end
+
+  def self.request_task_class_from(_attribs)
+    ManageIQ::Providers::Lenovo::PhysicalInfraManager
```

Comment:
This should not be the specific Lenovo provider here, we should reference the base class instead

Reply:
@rodneyhbrown7 It should actually be "ManageIQ::Providers::Lenovo::PhysicalInfraManager::ProvisionTask", so the correct provisioning task is created. Otherwise, the foreman provision task and associated state machine will be used. I have not pushed these changes up yet.